FirebaseRemoteConfig/Sources/RCNFetch.m

```diff
@@ -192,18 +190,24 @@ - (void)fetchConfigWithExpirationDuration:(NSTimeInterval)expirationDuration
                                          withError:error];
     }
     strongSelf->_settings.isFetchInProgress = YES;
-    [strongSelf refreshInstanceIDTokenAndFetchCheckInInfoWithCompletionHandler:completionHandler];
+    [strongSelf refreshInstallationsTokenWithCompletionHandler:completionHandler];
   });
 }
 
 #pragma mark - Fetch helpers
 
-/// Refresh instance ID token before fetching config. Instance ID is now mandatory for fetch
+- (NSString *)FIRAppNameFromFullyQualifiedNamespace {
+  NSString *FIRAppName =
+      [_FIRNamespace substringFromIndex:[_FIRNamespace rangeOfString:@":"].location + 1];
```

Comment:
I am a bit worried that we compose the string at one place and parse it at another, because it increases probability of errors when the logic needs to be updated. Will it be better to just pass `appName` separately? 

If you still would like to parse the string, I would suggest a safer way to do it to avoid crashing if the `_FIRNamespace` contains an unexpected string for some reason, e.g.:
```
return [[_FIRNamespace componentsSeparatedByString:@":"] lastObject];
```

Reply:
Done.